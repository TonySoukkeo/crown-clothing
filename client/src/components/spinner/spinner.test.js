import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import Spinner from "./spinner.component";

configure({
  adapter: new Adapter(),
});

describe("tests spinner component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Spinner />);
  });

  it("renders spinner component without crashing", () => {
    expect.assertions(1);
    expect(wrapper).toMatchSnapshot();
  });
});
