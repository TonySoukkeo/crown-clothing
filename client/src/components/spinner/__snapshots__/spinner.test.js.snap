// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests spinner component renders spinner component without crashing 1`] = `ShallowWrapper {}`;
